packages/components/time-select/__tests__/time-select.test.tsx

```diff
@@ -283,7 +283,7 @@ describe('TimeSelect', () => {
       )
     })
 
-    it('form item role is group when multiple inputs', async () => {
+    it('The disabled state of a component has higher priority than that of a form', async () => {
```

Comment:
```suggestion
    it('form item role is group when multiple inputs', async () => {
```

It seems this was changed in the wrong place.